### ISSUE TITLE
InfoBox hover colour

### ISSUE THREAD

**Issue by gestchild** [wellcometrust/wellcomecollection.org#3303]
References #3289

the link hover colour is not accessible on the yellow background

![info-link](https://user-images.githubusercontent.com/6051896/45306783-5cc2a180-b515-11e8-99d5-2c18c17dbd20.gif)

**Comment by GarethOrmerod** [comment]
Link style to be Black Underline as per https://zpl.io/b6v7QEm

**Comment by gestchild** [comment]
@GarethOrmerod That is the link style, it's the hover colour that is the issue. At the moment it does what other links do and turns green as above.
